android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -203,7 +203,7 @@
 		"greatPersonPoints": {"gold": 1},
 		"isWonder": true,
 		"providesFreeBuilding":  "Lighthouse",
-		"uniques": ["Must be next to [Coast]", "All military naval units receive +1 movement and +1 sight"],
+		"uniques": ["Must be next to [Coast]", "+[1] movement and +[1] sight for all [military water] units"],
```

Comment:
Split movement and sight into 2 separate uniques

Reply:
I think that they should be capitalized as well, I can't recall what we usually use